src/jit/morph.cpp

```diff
@@ -4826,9 +4862,10 @@ GenTreePtr Compiler::fgMorphMultiregStructArg(GenTreePtr arg, fgArgTabEntryPtr f
 
                     // Create a new tree for 'arg'
                     //    replace the existing LDOBJ(ADDR(LCLVAR))
-                    //    with a LIST(LCLVAR-LO, LIST(LCLVAR-HI, nullptr))
+                    //    with a FIELD_LIST(LCLVAR-LO, FIELD_LIST(LCLVAR-HI, nullptr))
                     //
-                    newArg = gtNewAggregate(hiLclVar)->Prepend(this, loLclVar);
```

Comment:
Why get rid of the `Prepend` method? It at least avoids the need for the `(void)` cast.